cmd/bosun/expr/funcs.go

```diff
@@ -1182,6 +1199,51 @@ func Rename(e *State, T miniprofiler.Timer, set *Results, s string) (*Results, e
 	return set, nil
 }
 
+// csv is a shorthand for strings.Split(s, ",")
+func csv(s string) []string {
+	return strings.Split(s, ",")
+}
+
+func MetaTag(e *State, T miniprofiler.Timer, set *Results, lookupTagsCSV, keyCSV string) (*Results, error) {
+	keys := csv(keyCSV)
+	res := Results{}
+	lookupTags := make(map[string]struct{})
+	for _, k := range csv(lookupTagsCSV) {
+		lookupTags[k] = struct{}{}
```

Comment:
I know `struct{}` is technically more performant or whatever, but bools are so much cleaner looking. If we're gonna do that, maybe we should define a `stringSet` type somewhere with proper `Contains` and `Put` methods. But just use map[string]bool